admin/tools/cluster/cc-in2p3/shmux/scripts/params.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/sh
+
+# LSST Data Management System
+# Copyright 2014-2015 LSST Corporation.
+#
+# This product includes software developed by the
+# LSST Project (http://www.lsst.org/).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
+# see <http://www.lsstcorp.org/LegalNotices/>.
+
+
+# Common value used by all cluster management scripts
+
+# @author  Fabrice Jammes, IN2P3/SLAC
+
+set -e
+
+################################################################
+#
+# Edit variables below to specify values for a given cluster
+#
+################################################################
+
+# Default values below are for CC-IN2P3 50 nodes cluster
+
+MASTER='ccqserv100.in2p3.fr'
+
+SHARED_DIR=/sps/lsst/Qserv/stack/
+
+INSTALL_DIR=/qserv/stack
+
+QSERV_RUN_DIR=/qserv/run
+
+QSERV_DATA_DIR=/qserv/data
+
+# Tag related to Qserv version built from sources,
+# has to be added to stack as a global eups tag available on the cluster
+# Declare this tag tag in $EUPS_PATH/ups_db/global.tags, if not exists:
+#
+# grep -q -F 'latestbuild' $EUPS_PATH/ups_db/global.tags || echo 'latestbuild' >> $EUPS_PATH/ups_db/global.tags
+#
+# and then rsync stack to all nodes.
+SETUP_OPT_GIT='-t latestbuild'
+
+SETUP_OPT_RELEASE='-t qserv'
+
+function check_path {
+
+    case "$1" in
+        /*) ;;
+        *) echo "expect absolute path for $1" ; exit 2 ;;
+    esac
+
+    # strip trailing slash
+    echo $1 | sed 's%\(.*[^/]\)/*%\1%'
```

Comment:
Is this needed at all, why are we sensitive to directory names having trailing slash?